File: app/src/main/java/com/makingiants/today/todayhistory/utils/log/DebugTree.java
package com.makingiants.today.todayhistory.utils.log;

import timber.log.Timber;

public class DebugTree extends Timber.DebugTree {

  @Override
  protected String createStackElementTag(StackTraceElement element) {
    // Print line number too
    return super.createStackElementTag(element) + ":" + element.getLineNumber();
  }
}

File: app/src/main/kotlin/com/makingiants/todayhistory/utils/log/DebugTree.kt
package com.makingiants.todayhistory.utils.log

import timber.log.Timber

class DebugTree : Timber.DebugTree() {

    override fun createStackElementTag(element: StackTraceElement): String {
        // Print line number too
        return super.createStackElementTag(element) + ":" + element.lineNumber
    }
}
